integration-tests/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+quarkus.native.additional-build-args=-H:DynamicProxyConfigurationResources=proxy-config.json
```

Comment:
This is redundant if you place `proxy-config.json` under `META-INF/native-image/<group-id>/<artifact-id>/`